src/plugins/NEMD/VelocityExchange.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+class VelocityExchangeTest;
+#include <string>
+#include <map>
+#include <unordered_map>
+#include <list>
+#include <cstdint>
+#include <vector>
+#include <array>
+
+#include "plugins/PluginBase.h"
+#include "molecules/Molecule.h"
+#include "particleContainer/ParticleContainer.h"
+#include "utils/CommVar.h"
+
+/** @brief Read in XML configuration for VelocityExchange
+ *
+ * The following XML object structure is handled by this method:
+ * \code{.xml}
+    <plugin name="VelocityExchange">
+        <control>
+            <start>0</start>                     <!-- step to start; default 0 -->
+            <frequency>10000</frequency>         <!-- frequency to conduct velo swap; default 10000 -->
+            <stop>200000000</stop>               <!-- step to stop; default 200000000 -->
+        </control>
+        <coldregion>                              <!-- region with lower temperature -->
+            <xmin>0</xmin> <xmax>box</xmax>      <!-- range x-axis; default: 0 to box size -->
+            <ymin>140</ymin> <ymax>160</ymax>    <!-- range y-axis -->
+            <zmin>0</zmin> <zmax>box</zmax>      <!-- range z-axis; default: 0 to box size -->
+        </coldregion>
+        <warmregion>                              <!-- region with higher temperature -->
+            <symmetric>1</symmetric>             <!-- 0: no symmetry; 1: symmetry in y direction (default) -->
```

Comment:
the doc of this parameter is not really clear to me. What is supposed to be symmetric? Position or size of the Box? Would this make x/y/z of one of the two regions irrelevant? Or is it about something else?

Reply:
I added some doc to clearify the purpose of the `<symmetric>` option.